docker-compose.yml

```diff
@@ -0,0 +1,90 @@
+version: '3'
+services:
+  postgres-db:
+    container_name: database
+    image: mdillon/postgis:9.5
+    restart: always
```

Comment:
I think that this should be `on-failure` or `unless-stopped` so that the container doesn't keep restarting itself even when stopped.